kubeproxy/README.md

```diff
@@ -0,0 +1,67 @@
+# kubeproxy-operator
+
+kubeproxy-operator is a Kubernetes operator for managing kubeproxy.
+
+## Running in a cluster
+
+1. Create a kinder cluster
+   Ensure kinder is installed. [Installation docs](https://github.com/kubernetes/kubeadm/blob/master/kinder/README.md)
```

Comment:
just a FYI, from the kinder docs:

> The exported logic in kinder packages can be a subject of change at any point in time, thus using kinder as a library is unsupported.

this message is rather incomplete.

one caveat is that kinder has no support guarantees for usage external to the kubeadm e2e tests even for its CLI. this means that if we have to change something in kinder we would likely do it without a deprecation policy and me might break consumers.

but overall using it for testing the operators seems fine and if one day kubeadm uses the operators we would likely run tests on such a setup using kinder.